sdk/webpubsub/azure-messaging-webpubsub/README.md

```diff
@@ -0,0 +1,179 @@
+# Azure Web PubSub client library for Java
+
+Azure Web PubSub client library for Java allows sending messages to Web PubSub. Azure Web PubSub service enables you to
+build real-time messaging web applications using WebSockets and the publish-subscribe pattern. Any platform  
+supporting WebSocket APIs can connect to the service easily, e.g. web pages, mobile applications, edge devices, etc. The
+service manages the WebSocket connections for you and allows up to 100K concurrent connections. It provides powerful
+APIs for you to manage these clients and deliver real-time messages.
+
+Any scenario that requires real-time publish-subscribe messaging between server and clients or among clients, can use
+Azure Web PubSub service. Traditional real-time features that often require polling from server or submitting HTTP
+requests, can also use Azure Web PubSub service.
+
+[Source code][source_code] | [Product Documentation][product_documentation] | [Samples][samples_readme]
+
+## Getting started
+
+### Prerequisites
+
+- A [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Azure Subscription][azure_subscription]
```

Comment:
`jdk_link` and `azure_subscription` not defined in below Links section

Reply:
Fixed.